<launch>
  <arg name="cloud1" default="velodyne_points_front"/>
  <arg name="cloud2" default="velodyne_points_rear"/>
  <arg name="cloud_out" default="cloud_merged"/>
  <arg name="target_frame" default="vehicle_1/base_footprint"/>

  <node pkg="pointcloud_concatenate" type="pointcloud_concatenate_node" name="cloud_concat" output="screen">
    <remap from="cloud1" to="$(arg cloud1)"/>
    <remap from="cloud2" to="$(arg cloud2)"/>
    <remap from="cloud_out" to="$(arg cloud_out)"/>
    <param name="target_frame" value="$(arg target_frame)"/>
  </node>
</launch>